ZConfig/tests/test_pygments.py

```diff
@@ -0,0 +1,221 @@
+##############################################################################
+#
+# Copyright (c) 2019 Zope Corporation and Contributors.
+# All Rights Reserved.
+#
+# This software is subject to the provisions of the Zope Public License,
+# Version 2.1 (ZPL).  A copy of the ZPL should accompany this distribution.
+# THIS SOFTWARE IS PROVIDED "AS IS" AND ANY AND ALL EXPRESS OR IMPLIED
+# WARRANTIES ARE DISCLAIMED, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF TITLE, MERCHANTABILITY, AGAINST INFRINGEMENT, AND FITNESS
+# FOR A PARTICULAR PURPOSE.
+#
+##############################################################################
+"""\
+Tests for ZConfig.pygments.ZConfigLexer.
+
+"""
+
+from __future__ import print_function, absolute_import
+
+import unittest
+
+try:
+    import pygments.lexer
+    import pygments.token
+except ImportError:
+    pygments = None
+
+import ZConfig.pygments
```

Comment:
This will try to import pygments.lexer again and raise the ImportError that will not be caught.

Reply:
Thanks, @mgedmin!  Fixed in 4bff45f.